pandas/tests/series/test_analytics.py

```diff
@@ -336,6 +336,25 @@ def test_describe(self):
                           index=['count', 'unique', 'top', 'freq'])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("tz", [None, "US/Eastern"])
+    def test_describe_dt(self, tz):
```

Comment:
Could you make the test name a bit more descriptive, like `test_describe_timezone_dtype`

Reply:
I don't quite think that's the right name since the test also tests tz-naive datetimes. How about `test_describe_dt_dtype`?